src/test/java/com/vite/service/ViteHandlerTest.java

```diff
@@ -1,16 +1,19 @@
 package com.vite.service;
 
 import org.junit.Test;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
 
 
 public class ViteHandlerTest {
     /**
      * Tests the {@link ViteHandler#ping()} function
-     * TODO add assertion about log statement, or better yet make ping return a String
      */
     @Test
     public void testPing() {
         ViteHandler viteService = new ViteHandler();
-        viteService.ping();
+        String response = viteService.ping();
+        String expected = "Server Response to ping at ";
+        assertThat(response.contains(expected), is(true));
```

Comment:
Because you're looking for english in your code.

If you're gonna do that, why not do `assertThat(response, containsString(expected))`? That would actually tell you what's wrong if it fails, instead of saying `Was 'false', expected 'true'`. But whatever.